code/modules/mob/living/simple_animal/vox.dm

```diff
@@ -22,15 +22,14 @@
 	var/amp = null
 	var/quills = 3
 
-/mob/living/simple_animal/vox/armalis/death()
-
-	living_mob_list -= src
-	dead_mob_list += src
-	stat = DEAD
-	visible_message("\red <B>[src] shudders violently and explodes!</B>","\red <B>You feel your body rupture!</B>")
-	explosion(get_turf(loc), -1, -1, 3, 5)
-	src.gib()
-	return
+/mob/living/simple_animal/vox/armalis/death(gibbed)
+	if(!gibbed)
+		stat = DEAD
+		visible_message("<span class='danger'>[src] shudders violently and explodes!</B>","\red <B>You feel your body rupture!</span>")
+		explosion(get_turf(loc), -1, -1, 3, 5)
+		gib()
+		return
+	return ..()
 
 /mob/living/simple_animal/vox/armalis/attackby(var/obj/item/O as obj, var/mob/user as mob, params)
```

Comment:
I would just like to add that Vox Armalis code is absolutely horrendous. When you attack one, the attack animation that plays for hitting just about anything else does not play, and at one point I was hitting one with a lit energy axe (force = 150) for testing, and it wasn't taking damage.